test/e2e/config/default.json

```diff
@@ -7,6 +7,7 @@
 	"startBrowserTimeoutMS": 60000,
 	"startAppTimeoutMS": 240000,
 	"afterHookTimeoutMS": 60000,
+	"playwrightTimeoutMS": 30000,
```

Comment:
This mirrors the value used by Playwright internally - 30000ms or 30s as the timeout.